environment/src/test/java/com/sequenceiq/environment/environment/poller/SdxPollerProviderTest.java

```diff
@@ -56,14 +60,22 @@ void testStartDatalakePoller(SdxClusterStatusResponse s1Status, SdxClusterStatus
         SdxClusterResponse sdx1 = getSdxResponse(s1Status, "crn1");
         SdxClusterResponse sdx2 = getSdxResponse(s2Status, "crn2");
 
-        Mockito.when(sdxService.getByCrn("crn1")).thenReturn(sdx1);
-        Mockito.when(sdxService.getByCrn("crn2")).thenReturn(sdx2);
+        when(sdxService.getByCrn("crn1")).thenReturn(sdx1);
+        when(sdxService.getByCrn("crn2")).thenReturn(sdx2);
 
         AttemptResult<Void> result = underTest.startSdxClustersPoller(ENV_ID, pollingCrn).process();
 
         assertEquals(attemptState, result.getState());
     }
 
+    @ParameterizedTest
```

Comment:
Nit: Are we happy with the `DEFAULT_DISPLAY_NAME` or should possibly provide a `name="..."` explicity?

Reply:
I was happy but I am happier now.